WebApplication1/Services/RoleService.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using WebApplication1.Configuration;
+
+namespace WebApplication1.Services
+{
+    public class RoleService
+    {
+        private readonly SocketGuild _guild;
+        public readonly Config Config;
+
+        public RoleService(Config config, DiscordSocketClient client)
+        {
+            Config = config;
+            _guild = client.Guilds.FirstOrDefault(x => x.Id == Config.GuildId);
+        }
+
+        public async Task AssignRoleAsync(ulong userId, IEnumerable<string> roles)
+        {
+            // TODO find the nullref bug
+            try
+            {
+                var rolesToAdd = new List<IRole>();
+                foreach (var x in roles)
```

Comment:
Try to not (ab)use `var` too much. If I were to take this one line out of context (for example when looking at diff) I wouldn't know what type is `x` here.

Reply:
Noted, I am being tempted by R# screaming to change everything to var :D Changed var to string and x to a proper name